html-forms/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <div>
+        <label for="email">Email</label>
+      </div>
+      <input type="text" id="email">
+      <div>
+        <label for="coding-cool">Why is coding cool?</label>
+      </div>
+      <select name="options">
+        <option value="Select an option">Select an option</option>
+        <option value="Because it's interesting">Because it's interesting</option>
+        <option value="Because it's fun">Because it's fun</option>
+        <option value="Because it's powerful">Because it's powerful</option>
+      </select>
+      <p>Favorite Language (choose one)</p>
+      <input type="radio" name="language" value="HTML"> HTML
+      <input type="radio" name="language" value="CSS"> CSS
+      <input type="radio" name="language" value="JavaScript"> JavaScript
+      <p>Would you like to sign up for more information?</p>
+      <input type="checkbox" name="signup" value="Sign me up!"> Sign me up!
+      <div>
+        <label>Comments</label>
```

Comment:
make sure to add the proper `for` attribute here and a matching id on the textarea